components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.viewport.js

```diff
@@ -1136,6 +1150,12 @@ jQuery._WeblitzViewport = function (container, server, options) {
         }
         if (t.length > 1) {
           t = t[1].split('$');
+          if (t[0].endsWith('-r')) {
+            this.setChannelReverseIntensity(idx, false, true);
+          } else if (t[0].endsWith('r')) {
+            this.setChannelReverseIntensity(idx, true, true);
+          }
+          t[0] = t[0].replace('-r', '').replace('r', '');  // remove 'r' if present
           var range = t[0].split(':');
           if (range.length == 2) {
```

Comment:
if you went with the proposal for ':' separation then it'd need to be

`if (range.length == 3)  
    this.setChannelReverseIntensity(idx, range[2].length > 0 && range[2] === 'r', true)`